module/VuFind/sql/mysql.sql

```diff
@@ -265,7 +265,7 @@ CREATE TABLE `user_list` (
   `title` varchar(200) NOT NULL,
   `description` text,
   `created` datetime NOT NULL DEFAULT '2000-01-01 00:00:00',
-  `public` int(11) NOT NULL DEFAULT '0',
+  `public` tinyint NOT NULL DEFAULT '0',
```

Comment:
I think this should be `tinyint(1)` for consistency with other tinyint values in the schema.